rpms/SPECS/3.11.4/wazuh-api-3.11.4.spec

```diff
@@ -0,0 +1,244 @@
+    systemctl disable wazuh-api.service > /dev/null
+    rm -f /etc/systemd/system/wazuh-api.service
+  fi
+
+  if [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+    /etc/init.d/wazuh-api stop > /dev/null
+    chkconfig wazuh-api off
+    chkconfig --del wazuh-api
+    rm -f /etc/rc.d/init.d/wazuh-api || true
+  fi
+fi
+
+%posttrans
+
+# Restart the Wazuh API service in case the installation was trigger as dependency
+if [ -n "$(ps -e | egrep ^\ *1\ .*systemd$)" ]; then
+  systemctl start wazuh-api.service > /dev/null
+elif [ -x /etc/rc.d/init.d/wazuh-api ] ; then
+  /etc/rc.d/init.d/wazuh-api start > /dev/null
+elif [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+  /etc/init.d/wazuh-api start > /dev/null
+else
+  echo "Error: Wazuh API could not restart"
+fi
+
+%clean
+rm -fr %{buildroot}
+
+%files
+%defattr(-,root,root)
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api
+%attr(750, root, ossec) %{_localstatedir}/ossec/api/app.js
+%attr(640, root, ossec) %{_localstatedir}/ossec/api/package.json
+%attr(644, root, root) %{_localstatedir}/ossec/api/package-lock.json
+%dir %attr(750, root, ossec) %config(noreplace) %{_localstatedir}/ossec/api/configuration
+%attr(740, root, ossec) %config(noreplace) %{_localstatedir}/ossec/api/configuration/config.js
+%attr(750, root, root) %{_localstatedir}/ossec/api/configuration/preloaded_vars.conf
+%dir %attr(750, root, root) %config(noreplace) %{_localstatedir}/ossec/api/configuration/auth
+%{_localstatedir}/ossec/api/configuration/auth/htpasswd
+%attr(660, root, root) %config(noreplace) %{_localstatedir}/ossec/api/configuration/auth/user
+%dir %attr(750, root, root) %config(noreplace) %{_localstatedir}/ossec/api/configuration/ssl
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/controllers
+%attr(750, root, ossec) %{_localstatedir}/ossec/api/controllers/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/examples
+%attr(750, root, ossec) %{_localstatedir}/ossec/api/examples/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/helpers
+%attr(750, root, ossec) %{_localstatedir}/ossec/api/helpers/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/models
+%attr(750, root, ossec) %{_localstatedir}/ossec/api/models/*
+%dir %attr(750, root, root) %{_localstatedir}/ossec/api/scripts
+%attr(750, root, root) %{_localstatedir}/ossec/api/scripts/*.sh
+%attr(640, root, root) %{_localstatedir}/ossec/api/scripts/wazuh-api
+%attr(640, root, root) %{_localstatedir}/ossec/api/scripts/wazuh-api.service
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules
+%attr(750, ossec, ossec) %{_localstatedir}/ossec/api/node_modules/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules/.bin
+%{_localstatedir}/ossec/api/node_modules/.bin/*
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/logs/api
+%attr(660, ossec, ossec) %ghost %{_localstatedir}/ossec/logs/api.log
+
+%changelog
+* Thu Jan 13 2020 support <info@wazuh.com> - 3.11.4
```

Comment:
Change this date please

Reply:
Done: 20b79cbda984df6fd487ae51afe52f2171648611